daemon/src/rollover/protocol.rs

```diff
@@ -232,22 +231,22 @@ pub enum CompleteFee {
     Nein,
 }
 
-impl From<FeeFlow> for CompleteFee {
-    fn from(fee_flow: FeeFlow) -> Self {
-        match fee_flow {
-            FeeFlow::LongPaysShort(a) => CompleteFee::LongPaysShort(a),
-            FeeFlow::ShortPaysLong(a) => CompleteFee::ShortPaysLong(a),
-            FeeFlow::Nein => CompleteFee::Nein,
+impl From<model::CompleteFee> for CompleteFee {
+    fn from(complete_fee: model::CompleteFee) -> Self {
+        match complete_fee {
+            model::CompleteFee::LongPaysShort(a) => CompleteFee::LongPaysShort(a),
```

Comment:
I am not sure that will make the code easier to read and understand, but feel free to refactor this at some point :)

Reply:
We had that in the beginning, didn't we? then we decided that it will be easier and less confusing to use this terminology. Happy to move back to positive and negative numbers, as long as we have a clear doc where we state explicitly what is what.

Reply:
> We had that in the beginning

IMHO this is only half of the truth - what we had initially was incomplete. That's why we introduced `FeeAccount` and other structs to manage this. I agree that changing the logic of`+` and `-` instead to an `enum` might mean more code, but I would be careful stating that it can easily be made "better" by using `+` and `-`. We re-wrote this code three times already. 

 I am happy to see another refactoring to this part as long as it has at least the same test coverage that we have now :)